src/elements/content-explorer/stories/tests/ContentExplorer-e2e.test.stories.js

```diff
@@ -0,0 +1,15 @@
+// @flow
```

Comment:
all new files should be using typescript

Reply:
Is there somewhere I can write that down as the convention?